turbinia/evidence.py

```diff
@@ -38,6 +41,65 @@
 log = logging.getLogger('turbinia')
 
 
+def evidence_class_names(all_classes=False):
+  """Returns a list of class names for the evidence module.
+
+  Args:
+    all_classes (bool): Flag to determine whether to include all classes
+        in the module.
+  """
```

Comment:
Maybe add a `Returns` section here just to be complete.

Reply:
Done